tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2802,7 +2802,58 @@ declare namespace ts {
                 readonly includeInlayFunctionLikeReturnTypeHints?: boolean;
                 readonly includeInlayEnumMemberValueHints?: boolean;
                 readonly autoImportFileExcludePatterns?: string[];
+                /**
+                 * Indicates whether imports should be organized in a case-insensitive manner.
+                 */
                 readonly organizeImportsIgnoreCase?: "auto" | boolean;
+                /**
+                 * Indicates whether imports should be organized via an "ordinal" (binary) comparison using the numeric value
+                 * of their code points, or via "unicode" collation (via the
+                 * [Unicode Collation Algorithm](https://unicode.org/reports/tr10/#Scope)) using rules associated with the locale
+                 * specified in {@link organizeImportsCollationLocale}.
+                 *
+                 * NOTE: When comparing paths `"ordinal"` collation is always used.
```

Comment:
Is this still accurate after https://github.com/microsoft/TypeScript/pull/52115/commits/72678a3dde6be15abe5c583b6e769dceac2073e0 ? (I can't really tell)

Reply:
No, I'll update.